import { getServerSession } from "next-auth"
import { authOptions } from "@/auth"
import { NextResponse } from "next/server"

export async function GET() {
  try {
    const session = await getServerSession(authOptions)
    return NextResponse.json(session)
  } catch (error) {
    console.error("Erro ao obter sess√£o:", error)
    return NextResponse.json({ error: "Erro interno do servidor" }, { status: 500 })
  }
}